CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # master
 
+# 0.3.4
+
+* Add source_id to `Account` and assign it when app is being installed. In order to make this work:
+
+1. Update your `Gemfile`
```

Comment:
sorry for the fuss, could we leave just a shot info here and move the whole instruction thing to some wiki entry